runtime/browser/xwalk_runner_win.cc

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2016 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "xwalk/runtime/browser/xwalk_runner_win.h"
+
+#include "base/win/registry.h"
+#include "xwalk/application/browser/application.h"
+#include "xwalk/application/browser/application_service.h"
+#include "xwalk/application/browser/application_system.h"
+
+namespace xwalk {
+
+namespace {
+
+const auto g_google_api = L"GOOGLE_API_KEY";
+const auto g_google_default_client_id = L"GOOGLE_DEFAULT_CLIENT_ID";
+const auto g_google_default_client_secret = L"GOOGLE_DEFAULT_CLIENT_SECRET";
+
+}  // namespace
+
+XWalkRunnerWin::XWalkRunnerWin() {
+}
+
+void XWalkRunnerWin::InitializeEnvironmentVariablesForGoogleAPIs(
+  content::RenderProcessHost* host) {
+  application::Application* app =
+      app_system()->application_service()->
+          GetApplicationByRenderHostID(host->GetID());
+  if (!app)
+    return;
+  std::string app_name;
+  if (!app->data()->GetManifest()->GetString("xwalk_package_id", &app_name))
```

Comment:
nit: pls add a const variable to xwalk/application/common/application_manifest_constants.h/cc for "xwalk_package_id"